### ISSUE TITLE
CSC matrix is treated as CSR

### ISSUE THREAD

**Issue by fverdugo** [gridap/GridapPardiso.jl#8]
We are passing the arrays to pardiso as though we had a CSR matrix, but we have a CSC one. 

As a result, the computed solution is wrong for non-symmetric matrices. 

We need to select `iparm[12] = 2`.

Here, we have an example leading into a non-symmetric matrix:

https://github.com/gridap/GridapPardiso.jl/blob/non_working_example/test/nonsymfemdriver.jl

Once the issue is solver, add this file to the test suite.

**Comment by victorsndvg** [comment]
fixed in 85d207b..de3573a

**Comment by victorsndvg** [comment]
merged in 4f2669d13aba742877ae104cea7387ed94ae052c

**Comment by fverdugo** [comment]
It would be interesting, to support the case where the user selects to solve the transpose, i.e., he/she writes `iparm[12]=2` in the driver.

I am thinking in updating the iparm with something like this:
```julia
if iparm[12]  in (0,1)
  iparm[12] = 2
elseif iparm[12] == 2
  iparm[12] = 1
else
 error("Unsuported iparm[12] == $(iparm[12]) for SparseMatrixCSC")
end
``` 

I reopen the issue

**Comment by victorsndvg** [comment]
fixed in 276a42b..20c2e1a

**Comment by victorsndvg** [comment]
merged in d0e462f2754ce179edaa8776dc1f31a8df0f59b2
